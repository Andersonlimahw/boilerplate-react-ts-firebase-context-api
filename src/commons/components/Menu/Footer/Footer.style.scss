footer {
    display: flex;
    align-items: center;
    justify-content: baseline;
    vertical-align: center;
    text-align: center;
    padding: 8px 16px;
    margin: 16px auto;
    color: #f1f1f1;
    border-top: 2px solid #c9c9c9;
}