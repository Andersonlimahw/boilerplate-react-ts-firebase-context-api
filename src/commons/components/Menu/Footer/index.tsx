import './Footer.style.scss';

export const Footer = () => {
    const moment = new Date().getFullYear();
    return(
        <footer> 
            Anderson Lima | {moment}
        </footer>
    )
}

export default Footer;