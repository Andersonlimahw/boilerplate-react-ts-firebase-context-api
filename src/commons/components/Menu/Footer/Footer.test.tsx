import React from 'react';
import { render, screen } from '@testing-library/react';
import Footer from './';

test('renders Footer component', () => {
  const { asFragment }  = render(<Footer />);
  const footerElement = screen.getByText(/Anderson Lima/i);
  expect(footerElement).toBeTruthy();
  expect(asFragment()).toMatchSnapshot();
});