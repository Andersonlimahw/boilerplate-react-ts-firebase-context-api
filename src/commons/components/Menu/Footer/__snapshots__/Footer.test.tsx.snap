// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Footer component 1`] = `
<DocumentFragment>
  <footer>
    Anderson Lima | 2023
  </footer>
</DocumentFragment>
`;

exports[`renders current year in Footer component 1`] = `
<DocumentFragment>
  <footer>
    Anderson Lima | 2023
  </footer>
</DocumentFragment>
`;
